app/coffee/infrastructure/Mongoose.coffee

```diff
@@ -2,6 +2,8 @@ mongoose = require('mongoose')
 Settings = require 'settings-sharelatex'
 logger = require('logger-sharelatex')
 
+mongoose.Promise = global.Promise
```

Comment:
The new version of mongoose issues a deprecation warning, about it's native Promises being deprecated and needing to set a promise implementation explicitly (http://mongoosejs.com/docs/promises.html)